src/components/breedingModal.html

```diff
@@ -58,7 +58,7 @@ <h5 class="modal-title" id="breedingModalLabel">Day Care</h5>
                                 <label>Regions</label><br/>
                                 <div class="dropdown show">
                                     <button type="button" class="text-left custom-select col-12 btn" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-bind="text :
-                                        BreedingFilters.region.value() == 2 ** (player.highestRegion() + 1) - 1 ?
+                                        BreedingFilters.region.value() || BreedingFilters.region.value() == 2 ** (player.highestRegion() + 1) - 1 ?
```

Comment:
As mentioned by Regis, this ternary always returns 'All' for any truthy region filter value now.
Maybe something like
```suggestion
                                        BreedingFilters.region.value() == undefined || BreedingFilters.region.value() == 2 ** (player.highestRegion() + 1) - 1 ?
```

Reply:
Actually, `undefined` is not the issue.
People stuck in the loading screen somehow kept the old value `-2` from before the update; but on a fresh new save...
It breaks because that can not be properly handled as a bitfield and results in inconsistent behaviors.